### PR TITLE
Provide an example of tour of heroes combining RxAngular with NGXS

### PR DESCRIPTION
**Pull request by Carniatto**
# Context

The objective of this PR is to provide a example of integrating RxAngular with NGXS using the best practices of both libraries.

### CODE REVIEW

File: apps/tour-of-heroes-ngxs/src/app/createPropertySelector.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { createSelector } from '@ngxs/store';
```
Comment (Carniatto) [inline apps/tour-of-heroes-ngxs/src/app/createPropertySelector.ts:1]: This is a utility for creating the basic selectors for your state. We are currently testing with some core users for refinement before adding to the library

Comment (BioPhoton) [reply apps/tour-of-heroes-ngxs/src/app/createPropertySelector.ts:1]: Ver nice!!!

**Comment by Carniatto** [comment]
Are there any tests I have to implement to validate this code and keep consistent with your workflow or we could merge if you think is good like this.

**Comment by BioPhoton** [comment]
Hi @Carniatto No all good.

Again a BIG thx for your effort!

**Comment by BioPhoton** [comment]
Cound you please remove the Draft status of your PR?

File: apps/tour-of-heroes-ngxs/src/app/hero-detail/hero-detail.component.ts [hunk 2]
```diff
@@ -27,21 +27,19 @@ export class HeroDetailComponent {
...
-        this.heroState.hero$(parseInt(params.get('id'), 10))
-      )
+      switchMap(params => this.heroState.hero$(parseInt(params.get('id'), 10)))
     );
     this.state.connect('hero', hero$);
     const saveEffect$ = this.save$.pipe(
       withLatestFrom(this.change$),
       map(([oldHero, change]) => ({ ...oldHero, ...change })),
-      switchMap((hero) => this.heroState.dispatchUpdateHero(hero)),
+      switchMap(hero => this.heroState.dispatchUpdateHero(hero)),
```
Comment (BioPhoton) [inline apps/tour-of-heroes-ngxs/src/app/hero-detail/hero-detail.component.ts:42]: You could also do something like this:

this.state.hold(saveTrigger$, hero => {
  this.heroState.dispatchUpdateHero(hero);
  this.goBack();
});

**Pull request merged**